content/influxdb/v2.0/reference/cli/influx/v1/auth/delete.md

```diff
@@ -0,0 +1,32 @@
+---
+title: influx v1 auth delete
+description: >
+  The `influx v1 auth delete` command deletes an authorization in the InfluxDB 1.x compatibility API.
+menu:
+  influxdb_2_0_ref:
+    name: influx v1 auth delete
+    parent: influx v1 auth
+weight: 101
+influxdb/v2.0/tags: [authorization]
+---
+
+The `influx v1 auth delete` command deletes an authorization in the [InfluxDB 1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+
+## Usage
+```
+influx v1 auth delete [flags]
+```
+
+## Flags
+| Flag |                   | Description                                                             | Input type | {{< cli/mapped >}}      |
+|:-----|:------------------|:------------------------------------------------------------------------|:----------:|:------------------------|
+| `-c` | `--active-config` | Config name to use for command                                          | string     | `$INFLUX_ACTIVE_CONFIG` |
+|      | `--configs-path`  | Path to the influx CLI configurations (default `~/.influxdbv2/configs`) | string     | `$INFLUX_CONFIGS_PATH`  |
+| `-h` | `--help`          | Help for the `delete` command                                           |            |                         |
+|      | `--hide-headers`  | Hide the table headers (defaults `false`)                               |            | `$INFLUX_HIDE_HEADERS`  |
+|      | `--host`          | HTTP address of InfluxDB                                                | string     | `$INFLUX_HOST`          |
+| `-i` | `--id`            | The authorization ID (required)                                         | string     |                         |
```

Comment:
Is the authorization ID different than the token referred to in `influx auth create` (...didn't see an auth ID over there)? If not the token, how can folks find this ID?

Reply:
Yes, the auth ID is the "parent" of a token. An authorization has a set of data associated with it (name, description, token). The reason the `influx auth create` command doesn't have an `--id` flag is because the auth doesn't exist yet and InfluxDB assigns the ID upon creation.